src/shared/firewall-util.h

```diff
@@ -8,7 +8,7 @@
 
 typedef struct FirewallContext FirewallContext;
 
-int fw_ctx_new(FirewallContext **ret);
+int fw_ctx_new(FirewallContext **ret, bool init_tables);
```

Comment:
here too it appears to me the old call should be retained, but the new one be called fw_ctx_full(). And the old call just just become a wrapper around the new call that passes the value in fixed

Reply:
OK

Reply:
Old name kept